spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -77,21 +81,24 @@
         data = json_response[:data]
 
         #Koiree: Check that the ID is correct and returned as a string.
-        expect(data[:id]).to eq(poster_id.to_s)
```

Comment:
After confirming the expected JSON formatting, we have added back in the '.to_s' to make :id a string